tests/integration/cocoon/test_cocoon_holdings.py

```diff
@@ -470,6 +514,7 @@ def test_load_from_data_frame_holdings_success(
             property_columns=property_columns,
             properties_scope=properties_scope,
             sub_holding_keys=sub_holding_keys,
```

Comment:
I'm getting a type suggestion warning here `Expected Optional[list], got 'dict' instead`. might be to do with holdings_adjustment_only missing in docstring (line 500 ish) for the test?

Reply:
Fixed, it was because we had

:param dict sub_holding_keys: The sub holding keys to populate on the adjustments as transaction properties

In the test docstring when it should have been

:param list sub_holding_keys: The sub holding keys to populate on the adjustments as transaction properties